bundles/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/MilightBinding.java

```diff
@@ -341,6 +335,11 @@ private PercentType sendIncrease(int bulb, int rgbwSteps, String bridgeId) {
             newValue = new PercentType(newPercent);
             logger.debug("milight: Bulb '{}' getting increased to '{}'", bulb, newPercent);
         }
+        try {
+            sendOn(bulb, bridgeId);
+            Thread.sleep(100);
+        } catch (InterruptedException e) {

```

Comment:
Is it correct to catch and "swallow" the InterruptedException (and below as well)?